src/stylesheets/base/_button.scss

```
@@ -0,0 +1,22 @@
+.button {
+  font-weight: 600;
+  font-size: 0.875rem;
+
+  &__primary {
```

Shouldn't it be this? 👀 

```suggestion
  &--primary {
```

---

yes. That make sense. It is the modifier. Button should be the element and anything that changes it as modifier. I was confused on it initially. Now I think about it should be modifier.